modules/learn/pages/services-and-indexes/services/services.adoc

```diff
@@ -1,99 +1,119 @@
+If a service becomes resource-constrained, you can add it to more nodes in the cluster to increase its capacity and meet demand.
 
-Service allocation should be designed based on workload-analysis:
-if a particular service is expected to handle a heavy workload,
-it should be allocated with a larger memory quota, and potentially as the only service on the node.
-Alternatively, if a cluster is to be used for development only,
-it may be convenient to allocate services in the quickest and most convenient way,
-with some quotas being equal.
+When you allocate a service to a node, you must assign it a memory quota.
+This quota becomes the standard for that service across all its instances in the cluster.
+The Query and Backup services are exceptions and do not require a memory quota.
 
-For example, the following illustration shows how four services — Data, Index, Query, and Search — might be allocated evenly across the five nodes of a _development_ cluster:
+Design the service allocation based on workload analysis.
+If a service is expected to handle a heavy workload, assign it a larger memory quota and consider running it as the only service on the node.
+For development clusters, you can allocate services quickly and conveniently, with some quotas set equally.
+
+The following example shows how four services such as Data, Index, Query, and Search that can be evenly allocated across five nodes in a development cluster.
 
 [#cb_cluster_with_services_development]
 include::partial$cluster-example-diagrams.adoc[tag="five-cluster-setup"]
 
-This configuration might provide perfectly acceptable performance for each of the four services, in the context of development and testing.
-However, if a large amount of data is required to be intensively indexed and addressed by means of Query and Search, the following production configuration would be more efficient:
+This configuration may provide adequate performance for development and testing.
+However, if large volumes of data require intensive indexing and querying with Search, the following production configuration is more efficient.
 
 [#cb_cluster_with_services_production]
 include::partial$cluster-example-diagrams.adoc[tag="six-cluster-setup"]
 
-In this revised configuration, the Data Service is the only service to run on three of the nodes;
-the Index Service is running on two further nodes; and the Query and Search Services share the sixth and final node.
+In the revised configuration, the Data Service runs exclusively on three nodes, the Index Service runs on two nodes, and the Query and Search services share the sixth node.
+
+For more information about:
+
+* Service memory quotas, see xref:buckets-memory-and-storage/memory.adoc[Memory].
+
+* Initializing a cluster, see xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
+
+* Adding or removing a Data Service on a node, see xref:manage:manage-nodes/manage-data-service-and-rebalance.adoc[Manage a Data Service and Rebalance].
```

Comment:
* Adding or removing a Data Service on a node, see xref:manage:manage-nodes/manage-data-service-and-rebalance.adoc[Adding or Removing the Data Service on existing nodes].